src/pages/storage/StorageList.tsx

```diff
@@ -130,29 +137,38 @@ const StorageList: FC = () => {
             />
           )}
           {!isLoading && !hasStoragePools && (
-            <EmptyState
-              iconName="pods"
-              iconClass="empty-storage-icon"
-              title="No storage found"
-              message="There are no storage pools in this project."
-            >
-              <>
-                <p>
-                  <a
-                    href="https://linuxcontainers.org/lxd/docs/latest/explanation/storage/"
-                    target="_blank"
-                    rel="noreferrer"
-                  >
-                    Learn more about storage
-                    <Icon className="external-link-icon" name="external-link" />
-                  </a>
-                </p>
-                <AddStorageBtn
-                  project={project}
-                  className="empty-state-button"
-                />
-              </>
-            </EmptyState>
+            <Row className="empty-state">
+              <Col size={6} className="col-start-large-4">
```

Comment:
I have given up this `Row` and `Col` approach in Anbox because, below the large breakpoint, it was causing the `EmptyState` to "collapse" towards the side nav instead of staying centred in the page:

![Peek 2023-08-09 15-07](https://github.com/canonical/lxd-ui/assets/56583786/8848a2be-0db0-48b6-a164-5ba982ec5eaa)

Check the `.empty-state` CSS class in Anbox for an alternative way to do this.

Reply:
The Row and Col approach is used to make the EmptyState centred, do we want it to be left un-centred?